src/dns/aardvark.rs

```diff
@@ -106,14 +106,17 @@ impl Aardvark {
             false => Stdio::null(),
         };
 
+        // After https://github.com/containers/aardvark-dns/pull/148 this command
+        // will block till aardvark-dns's parent process returns back.
         Command::new(&aardvark_args[0])
             .args(&aardvark_args[1..])
             .stdin(Stdio::null())
             .stdout(Stdio::null())
             .stderr(output)
```

Comment:
We should pass the fds to aardvark and let aardvark close(dup them to /dev/null) when it is ready

Reply:
Sure we could do that.

Reply:
Added in latest push and tested both the PR with podman